templates/attend.html

```diff
@@ -23,25 +25,18 @@ <h1>Attend the conference</h1>
             <div class="content">
                 <h2>Health, Safety and Accessibility</h2>
                 <div>
-                    <h3>Masks are required</h3>
-                    <img class="float-end" src="/images/masked-audience-2024.avif" width="500" height="380" alt="">
-                    <p>You need to wear a mask during your time at the Matrix Conference. That includes sessions,
-                        keynotes, booths, open spaces, and sprints. Exceptions are:
+                    <h3>Dietary Restrictions</h3>
+                    <p>All the food provided at the Matrix Conference will accommodate a wide range of dietary
+                        restrictions. All foods will be labelled at the event. If you have food allergies,
+                        please reach out to us
+                        at <a href="mailto:conference@foundation.matrix.org">conference@foundation.matrix.org</a>.
                     </p>
-                    <ul>
-                        <li>Outdoor spaces</li>
-                        <li>Indoors while actively consuming food or beverages</li>
-                        <li>Speakers while presenting</li>
-                        <li>While necessary for communicating with someone who is hearing impaired when the ability to
-                            see the mouth is essential for communication</li>
-                        <li>Very briefly to take photos</li>
-                    </ul>
-                    <p>Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable
-                        material. N95, FFP2 or equivalent masks are recommended. Alternatives to face coverings such as
-                        face
-                        shields, loose gaiters, scarves, bandanas, and face coverings with ventilation are not permitted
-                        to be
-                        used as masks. The Foundation will supply masks on-site.</p>
```

Comment:
we could still offer a (smaller) amount of free masks at the registration desk for anyone who feels unsafe, though I expect that it will not receive a lot of usage.

Reply:
I'm not too convinced that's useful, and I'd rather avoid having to carry extra stuff that doesn't get used